run/force_sync.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -eu
+
+printf "Forcing archiveloop to sync by pretending to take archive host offline and back online.\n"
+printf "Setting archive unreachable.."
+if ! touch /tmp/archive-is-unreachable; then
+   printf "Something went wrong!  Error %d\nAborting.\n" "$?" 1>&2
+   exit 1
+fi
+while [[ -f /tmp/archive-is-unreachable ]]; do
+   printf "Waiting for archiveloop to see canary file.."
+   sleep 5
+done
+printf "Setting archive reachable.."
+if ! touch /tmp/archive-is-reachable; then
```

Comment:
"then" on the next line

Reply:
Also, this doesn't seem like a good idea, since it tells archiveloop that the archive is reachable even when it isn't. It's better to just let archiveloop discover whether the archive is reachable on its own.

Reply:
That was my initial thought, but when I was using these canary files to test (and the archive _was_ reachable), it never seemed to get out of the 'unreachable' state until I dropped the new canary file.  I didn't look at the scripts, but I just shrugged and presumed that if you force the unreachable state, that you _had to_ force reachable again.  Also, if you force reachable when it really isn't, isn't this already handled by the existing tooling?

Reply:
> That was my initial thought, but when I was using these canary files to test (and the archive _was_ reachable), it never seemed to get out of the 'unreachable' state until I dropped the new canary file. I didn't look at the scripts, but I just shrugged and presumed that if you force the unreachable state, that you _had to_ force reachable again.

That's not the case. The script will detect that it is reachable if in fact it is reachable.

> Also, if you force reachable when it really isn't, isn't this already handled by the existing tooling?

Yes, in the sense that it'll try to archive and then fail because the archive server isn't actually reachable, however note that this script won't do the right thing if the archive is in fact NOT reachable, since in that case archiveloop will be waiting inside of wait_for_archive_to_be_reachable, and won't be looking for /tmp/archive-is-unreachable, let alone deleting it, so this script will then block until you get within reach of the archive server again.

Reply:
Very fair point.  Best, then, to let it occur naturally.